app/views/travel/index.html.erb

```diff
@@ -1,2 +1,5 @@
 <h1>Travel#index</h1>
-<p>Find me in app/views/travel/index.html.erb</p>
+
+<% @arrayOfGifs.each do |gif| %>
+  <%= image_tag("#{gif.view}?key=" + ENV["GOOGLE_KEY"]) %>
```

Comment:
It didn't work that magically, we might need to give a look to https://developers.google.com/drive/v3/web/picker